openpmd_viewer/openpmd_timeseries/data_reader/h5py_reader/field_reader.py

```diff
@@ -185,8 +185,17 @@ def read_field_circ( filename, iteration, field, coord,
     group, dset = find_dataset( dfile, iteration, field_path )
 
     # Extract the metainformation
-    Nm, Nr, Nz = get_shape( dset )
-    info = FieldMetaInformation( { 0: 'r', 1: 'z' }, (Nr, Nz),
+    coord_labels = {ii: coord.decode() for (ii,coord) in 
```

Comment:
Technically we would need to apply `dataOrder` here in openPMD 1, but we remove this in openPMD 2 because it is adding confusion without a useful degree of freedom.